cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -98,8 +96,13 @@ func (p *BicepProvider) Initialize(ctx context.Context, projectPath string, opti
 	if err := tools.EnsureInstalled(ctx, requiredTools...); err != nil {
 		return err
 	}
-
-	return p.EnsureEnv(ctx)
+	if options.BicepADS {
```

Comment:
NIT: perhaps just `p.ignoreADS = options.BicepADS`?  It will do the right thing in both cases without the if.